code/modules/reagents/machinery/chem_master.dm

```diff
@@ -452,14 +471,14 @@ var/global/list/pillIcon2Name = list("oblong purple-pink", "oblong green-white",
 	user.set_machine(src)
 
 	var/dat = list()
-	if(!beaker)
+	if(!beaker )
 		dat += "Please insert a beaker.<BR>"
 		if(!condi)
 			if(src.loaded_pill_bottle)
 				dat += "<A href='?src=\ref[src];ejectp=1'>Eject Pill Bottle \[[loaded_pill_bottle.contents.len]/[loaded_pill_bottle.storage_slots]\]</A><BR><BR>"
 			else
 				dat += "No pill bottle inserted.<BR><BR>"
-			dat += generate_pill_icon_div()
+			//dat += generate_pill_icon_div()
```

Comment:
This is intended so you can set the pill icon while no beaker is inserted to recolor timerelease pills and such

Reply:
readded